public/app/plugins/datasource/loki/responseUtils.ts

```diff
@@ -220,7 +220,7 @@ function cloneDataFrame(frame: DataQueryResponseData): DataQueryResponseData {
     ...frame,
     fields: frame.fields.map((field: Field<unknown, ArrayVector>) => ({
       ...field,
-      values: new ArrayVector(field.values.buffer),
```

Comment:
At runtime, `.buffer` will still work -- but the typing is not happy unless we also add `.buffer` to the core Array type.

Under the hood, `.buffer` and `.toArray()` are functionally equivalent -- so we can just add that to our change log notes 🤷

Reply:
I can't see you adding `buffer` to native Array type in this PR 🤔

Reply:
Any existing code that references ".buffer" is created as an ArrayVector and has a .buffer property.  However, with the new typing it is `T[] | ArrayVector<T>` and that *may* have a .buffer.  Runtime we are fine, the challenge is compile typecheck.  Our options here are either:
 1. Expand Array to have `.buffer` access
 2. Tell developers to replace .buffer with `toArray()` 

As I write that down, it seems like option 1 may be better 🤣

Reply:
adding `buffer` to Array has the unfortunate side-effect of infecting all Vectors with this property since the top-level `interface Vector<T> extends Array<T>`. we could maybe do something like `interface Vector<T> extends <Omit, Array<T>, 'buffer'`?